CRITICAL - 2020-11-03 07:36:40 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-03 07:37:43 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-03 07:37:48 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-03 07:37:50 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-03 07:37:50 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-03 07:37:51 --> Table 'rosas.rs_view_users' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
